slice/api/v1alpha1/slice_types.go

```diff
@@ -21,15 +21,25 @@ import (
 )
 
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
```

Comment:
It looks like this note related to Type. Can we add some space between to avoid confusion?